DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Ranking/Index.cshtml

```diff
@@ -35,7 +35,7 @@
     }
 
     @if (Model.CentreHasNoActivity) {
-      <p class="current-centre nhsuk-body-l nhsuk-u-padding-left-2">Your centre overall rank: no activity</p>
+      <p class="current-centre nhsuk-body-l nhsuk-u-padding-left-2" aria-current="true">Your centre overall rank: no activity</p>

```

Comment:
This looks good - do you think we could add a little bit of padding to the "Your centre overall rank: no activity" bit - maybe similar top/bottom padding as the table cells?